src/styles/index.scss

```
@@ -1,3 +1,20 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./card.scss";
+@import "./stars.scss";
+@import "../utils/variables";
+
+* {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
```

Only `box-sizing` is aloved to be in `*` selector
```suggestion
* {
  box-sizing: border-box;
}

html {
  font-family: Roboto, sans-serif;
}
```